### DISCUSSION TITLE
endpoint is always empty

### DISCUSSION THREAD

**Discussion by mitht** [apache/skywalking#11831] [Q&A]
oap 9.6
agent 9.0
springboot  2.6.6

skywalking endpoint is always empty; there is no request URL for springMvc; only database information

<img width="1830" alt="image" src="https://github.com/apache/skywalking/assets/29425860/ac38af5b-b97e-41df-bc5d-13b86ade48ee">

**Comment by wu-sheng** [comment]
No span for SpringMVC, then no relative endpoint, which is generated from MVC tracing.
